helm/azure-ad-pod-identity-app/templates/localredirectpolicy.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Values.provider (eq .Values.provider "capz") -}}
```

Comment:
how about simply checking if the CRD exists?

Reply:
I thought we did not want to install it on vintage, but if we do then sure i can do that :+1:

Reply:
I mean, if azure vintage will ever run cilium, then we need this so why not I guess

Reply:
make sense, also means i don't have to update config repo :+1: